syntax = "proto3";

package hello;

service VerificationService {
    rpc VerificationLogic (VerificationRequest) returns (VerificationResponse);
}

message VerificationRequest {
    repeated map<string, string> items = 1;
}

message VerificationResponse {
    string verdict = 1;
}